tfhe/src/integer/server_key/radix_parallel/scalar_div_mod.rs

```diff
@@ -135,6 +135,7 @@ pub trait SignedReciprocable:
     type DoublePrecision: DecomposableInto<u8>
         + ScalarMultiplier
         + CastFrom<<Self::Unsigned as Reciprocable>::DoublePrecision>
+        + CastInto<u64>
```

Comment:
Do we want to modify CPU's code?

Reply:
I have to add it to implement scalar_mul_high actually.